examples/configuration/echo.lua

```diff
@@ -0,0 +1,7 @@
+local policy_chain = require('apicast.policy_chain').build({ })
```

Comment:
I think these 2 lines could be simplified to:
`local policy_chain = require('apicast.policy_chain').build({'apicast.policy.echo'})`

Reply:
Yes. I wanted to show the `:insert` as an example. But that would be better with the default chain.

Reply:
What about:

```lua
local policy_chain = require('apicast.policy_chain').default()
local echo = require('apicast.policy.echo').new({ exit = 0 })

policy_chain:insert(echo, 1)
```

 ? 

It shows off the insert - how to extend the default chain and how to pass options to policy.

Reply:
But `apicast.policy.echo` doesn't have a constructor that can receive that table.

Reply:
Well I'd change it to accept a configuration for sake of the example.

Reply:
OK. In that case, I think that the latest example looks better than the original one :+1:

Reply:
Fixed in https://github.com/3scale/apicast/pull/496/commits/02ad4ab34e95041f9b288c73b27565d845aae123 and going to be squashed.